public/r2/v1/swagger/swagger.json

```diff
@@ -198,28 +198,29 @@
                     "application/json"
                 ],
                 "parameters": [
-                    {
-                        "in": "path",
-                        "name": "namespaceID",
-                        "description": "The name of the namespace",
-                        "type": "string",
-                        "required": true
-                    },
                     {
                         "in": "body",
                         "name": "ruleset-changes",
                         "description": "The updates to the ruleset",
                         "required": true,
                         "schema": {
-                        "$ref": "#/definitions/RuleSetChanges"
+                            "type": "object",
+                            "properties": {
+                                "ruleset-changes": {
+                                    "$ref": "#/definitions/RuleSetChanges"
+                                },
+                                "ruleset-version": {
+                                    "type": "integer"
+                                }
+                            }
                         }
                     },
                     {
-                        "in": "body",
-                        "name": "ruleset-version",
-                        "description": "The ruleset version to atomically apply bulk updates against",
-                        "required": true,
-                        "type": "integer"
+                        "in": "path",
```

Comment:
Hmm I dont think so. It may be the case that github is just making this hard to look at. What is actually happening here is that there were two entries for where `in: body` was present, which is not valid swagger. Also the body params must come first, so I moved the body params to the top of the list and merged the two elements in the list togther. I would suggest taking a look [here](https://github.com/m3db/m3ctl/blob/5a1720261bfcba373fd35fc088728e4c699d6b5c/public/r2/v1/swagger/swagger.json), it might be easier to see what is going on.

Reply:
Ah my bad - mislead by the UI